app/models/concerns/smooch_search.rb

```diff
@@ -198,13 +198,29 @@ def save_locally_and_return_url(media_url, type, feed_id)
         audio: 'audio/ogg',
         video: 'video/mp4'
       }[type.to_sym]
+      path = "feed/#{feed.id}/#{SecureRandom.hex}"
+      self.write_file_to_s3(
+        media_url,
+        path,
+        mime,
+        false,
+        headers
+      )
+    end
+    
+    def write_file_to_s3(media_url, path, mime, force_ssl, headers)
       uri = URI(media_url)
       http = Net::HTTP.new(uri.host, uri.port)
-      http.use_ssl = uri.scheme == 'https'
+      if force_ssl
+        http.use_ssl = true
+      else
+        http.use_ssl = uri.scheme == 'https'
+      end
```

Comment:
@DGaffney , I suggest a simplification here: you can replace this whole block by just `http.use_ssl = uri.scheme == 'https'`, and then we can also remove one argument for this method. I understand your approach here based on the previous code, but we can safely make this simplification.

Reply:
@caiosba done and done!